pyfar/signals/stochastic.py

```diff
@@ -95,26 +87,29 @@ def pulsed_noise(n_pulse, n_pause, n_fade=90, repetitions=5, rms=1,
         The default is 90, which equals approximately 2 ms at sampling rates of
         44.1 and 48 kHz.
     repetitions : int, optional
+        Specifies the number of noise pulses. The default is 5.
     rms : double, array like, optional
         The RMS amplitude of the white noise signal. The default is 1.
```

Comment:
```suggestion
        The RMS amplitude of the noise signal. The default is 1.
```

Reply:
Yes thanks - I initally had separate functions for white and pink noise. That looks like a leftover...